<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="game" native="true">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>480</width>
      <height>540</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QLabel" name="label_bg">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>480</width>
       <height>540</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="player">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="blocks">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>480</width>
       <height>540</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="endpoint" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>480</width>
        <height>540</height>
       </rect>
      </property>
      <widget class="QLabel" name="end1">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="end2">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>180</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="end3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>240</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="end4">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>360</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="end5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>360</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="end6">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>360</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="end7">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>360</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="movable" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>480</width>
        <height>540</height>
       </rect>
      </property>
      <widget class="QLabel" name="box1">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="box2">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>180</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="box3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>240</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="box4">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>360</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="box5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>360</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="box6">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>360</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="box7">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>360</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="finish_pic">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>170</y>
         <width>200</width>
         <height>120</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="btns" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>110</width>
      <height>150</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <widget class="QPushButton" name="NextlevelBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>90</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Next Level</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RestartBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>90</width>
       <height>50</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="text">
      <string>Restart</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="congrates">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Congratulations!&lt;/p&gt;&lt;p&gt;you pass the game&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>btns</zorder>
   <zorder>game</zorder>
   <zorder>congrates</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
